tools/release/sync-stable-blocks.js

```diff
@@ -50,7 +50,7 @@ ${ staticBlockFolderNames }
 		.map( toDirectoryName )
 		.sort()
 		// To PHP require_once statement:
-		.map( dirname => `require_once ABSPATH . WPINC . '/blocks/${ dirname }.php';` )
+		.map( dirname => `require_once BLOCKS_PATH . '${ dirname }.php';` )
```

Comment:
@vishalkakadiya this script use for refresh the packages in `Gruntfile.js`. Could you please share steps how anyone test it?